ultravox/tools/ds_tool/ds_tool.py

```diff
@@ -20,8 +20,7 @@
 @dataclasses.dataclass
 class TtsTask:
     implementation: str = simple_parsing.field(default="azure", alias="-i")
-    # Column name containing the text to convert to audio. It can be a Jinja variable expression.
```

Comment:
nit: why remove the comment?

Reply:
https://github.com/fixie-ai/ultravox/pull/69#discussion_r1704781138


Apprently it refers to code that no longer exists

Reply:
You're right, it should say something like: "Jinja template for the text that needs to be converted to audio".